applications/IgaApplication/tests/test_IgaApplication.py

```diff
@@ -31,6 +33,7 @@ def AssembleTestSuites():
         ScordelisRoofShell3pTest,
         TTestModelers
         ]))
+    smallSuite.addTests(KratosUnittest.TestLoader().loadTestsFromTestCases([TTestNurbsVolumeElements]))
```

Comment:
```suggestion
        ScordelisRoofShell3pTest,
        TTestNurbsVolumeElements,
        TTestModelers
        ]))
```

I reckon it is more readable in that way.